packages/terra-tabs/src/common-tabs/_Tabs.jsx

```diff
@@ -143,6 +179,19 @@ class Tabs extends React.Component {
         return { visibleTabData: newArray };
       });
     }
+
+    if (currTab && this.addButtonToggle && !this.moreButtonRef.current) {
```

Comment:
```suggestion
    if (currTab && !this.moreButtonRef.current) {
```
add button toggle is not required as already have it below condition before focus

Reply:
Done https://github.com/cerner/terra-framework/commit/628052153b88461ef88733be6b05c1fb519e3f41